### PR TITLE
LPS-183075 Review page definition for jobzz site initializer

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-183075)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

File: modules/apps/site-initializer/site-initializer-jobzz/src/main/resources/site-initializer/layouts/1_home/page-definition.json [hunk 1]
```diff
@@ -342,21 +238,15 @@
 											},
 											"displayAllItems": false,
 											"displayAllPages": false,
-											"listItemStyle": "",
-											"listStyle": "",
 											"numberOfColumns": 1,
 											"numberOfItems": 5,
 											"numberOfItemsPerPage": 5,
-											"numberOfPages": 5,
```
Comment (victorg1991) [inline modules/apps/site-initializer/site-initializer-jobzz/src/main/resources/site-initializer/layouts/1_home/page-definition.json:350]: We probably need this

File: modules/apps/site-initializer/site-initializer-jobzz/src/main/resources/site-initializer/layouts/1_home/page-definition.json [hunk 2]
```diff
@@ -647,30 +459,22 @@
 													},
 													"displayAllItems": false,
 													"displayAllPages": false,
-													"listItemStyle": "",
-													"listStyle": "",
 													"numberOfColumns": 1,
 													"numberOfItems": 2,
 													"numberOfItemsPerPage": 5,
-													"numberOfPages": 5,
```
Comment (victorg1991) [inline modules/apps/site-initializer/site-initializer-jobzz/src/main/resources/site-initializer/layouts/1_home/page-definition.json:655]: same here

**Comment by victorg1991** [comment]
Hey @DiegoHu97 I left some minor comments, could you take a look? :)

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
niiice!

**Comment by DiegoHu97** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by Tim-Cao** [comment]
ci:forward:force

**Comment by victorg1991** [comment]
ci:forward:force

**Pull request closed without merging**